src/Support/Authentication.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Nuwave\Lighthouse\Support;
+
+use Illuminate\Support\Facades\Auth;
+
+class Authentication
+{
+    /**
+     * @return \Illuminate\Config\Repository|\Illuminate\Contracts\Foundation\Application|\Laravel\Lumen\Application|mixed
```

Comment:
That return type seems way to imprecise.

Reply:
If you're right. As I see it it should be string | null, but even changing it to int | string it still gives error in phpstan. I don't really understand it very well.